spec/features/welcome_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+describe 'logging in' do
+  it 'as a registered user, I can log in with correct credentials' do
+    user = User.create(email: 'test123@xyz.com', password: 'viewparty')
+
+    visit '/'
```

Comment:
Just flagging again for my path helpers question

Reply:
See above response, thanks.